Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrkProducerFilter_cff.py

```diff
@@ -4,7 +4,7 @@
 #AlCaReco filtering for HCAL isotrk:
 #------------------------------------------------
 
-from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkProducer_cfi import *
```

Comment:
Why is this changed? The .cfi should be used, and it should be created by the fillDescribtion method in the producer .cc file

Reply:
There are some parameters which depend on the run period. The condition of EE changed over years due to radiation damage. So we need to modify some of the parameters which are ERA dependent. So all parameters are defined through the cfi created by fillDescription and the ERA dependence is put in the cff file which started from the cfi file.